lark/ast_utils.py

```diff
@@ -19,15 +19,17 @@ class AsList(object):
     Subclasses will be instanciated with the parse results as a single list, instead of as arguments.
     """
 
-def camel_to_snake(name):
-    return re.sub(r'(?<!^)(?=[A-Z])', '_', name).lower()
+class WithMeta(object):
+    """Abstract class
 
-def _call(func, _data, children, _meta):
-    return func(*children)
+    Subclasses will be instanciated the Meta instance of the tree. (see ``v_args`` for more detail)
```

Comment:
This is not a correct sentence I think.

Reply:
Also, at what position will it be provided?

Reply:
The same position `v_args` does. Which means first argument, but second if using `AsList`.

And yes, it's not ideal. I plan to switch them around for v1.0 so meta is always first.

Reply:
Why isn't it correct?

Reply:
I think there is at least a `with` missing

Reply:
Right, I missed that :)

Thanks!